### PR TITLE
i1205 - build jekyll pages without using github

### PR DESCRIPTION
**Pull request by richfitz**
This builds the pages on teamcity.  We'll still need to inject the results of successful builds into the running montagu instance but that will be a separate PR

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -ex
+IMAGE=docker.montagu.dide.ic.ac.uk:5000/vimc-website-builder:i1205
+docker pull $IMAGE
+VOLUME=$1
+BRANCH=$2
+if [ -z $VOLUME ]; then
+    VOLUME_MAP=""
```
Comment (hillalex) [inline build.sh:8]: should this mount the local "static" directory if a volume is not passed in?

**Review by hillalex — CHANGES_REQUESTED** [review]
When I ran `build.sh` locally, no files were created!

File: build.sh (hunk 1 above)
Comment (richfitz) [reply build.sh:8]: Did you try `./build.sh .` (see the note in the README)

Comment (hillalex) [reply build.sh:8]: that's what I ran...

Comment (hillalex) [reply build.sh:8]: oh no, sorry, duh!

**Review by hillalex — APPROVED** [review]

**Pull request merged**